documentation/profile-migration-guide.md

```diff
@@ -0,0 +1,33 @@
+# Guide for migrating from `profiles/**`
+
+This document is intended for users that are familiar with the previous version of the Azure SDK For Go profiles package (`github.com/Azure/azure-sdk-for-go/profiles/**`) and wish to migrate their application to the next version of Azure SDK for Go.
```

Comment:
... the older versions of Azure Profile SDK module for Go, and target to migrate applications to the latest version of the same module.

Reply:
The previous version does not use module. I changed some words accordingly.